packages/node/test/unit/helpers/weightedSample.test.ts

```diff
@@ -3,6 +3,7 @@ import { range, repeat, sum } from 'lodash'
 
 describe(weightedSample, () => {
     it('returns undefined on empty array', () => {
+        // eslint-disable-next-line @typescript-eslint/no-confusing-void-expression
```

Comment:
Why does this line need to be ignored?

Reply:
Added type generics. Without it, the generics type `T` evaluated to void.